README.md

```diff
@@ -169,14 +191,19 @@ If you want to just change one element of a default implementation, they aren't
 their methods are `virtual` so have at it. For instance, the `CustomValidationResultHandler` above might want to
 modify the way error codes are set on the `IError`, which can be done like so:
 
-```c#
+<!-- snippet: CustomValidationResultHandler -->
+<a id='snippet-customvalidationresulthandler'></a>
+```cs
 public class CustomValidationResultHandler : DefaultValidationResultHandler
 {
-    protected override IErrorBuilder ExtractError(IMiddlewareContext context, ValidationFailure failure)
-        => base.ExtractError(context, failure)
-            .SetExtension(nameof(failure.ErrorCode), $"CustomPrefix:{failure.ErrorCode}");
+    //TODO: what to do here since ExtractError doesnt exist
```

Comment:
note that this change highlighted a problem with the existing readme. 

suggestions on what should be done here?

Reply:
Hmm, good spot! I ended up with an ErrorFilter handling this instead and `DefaultValidationResultHandler` is currently a no-op. Figured it might be useful for logging if folks cared, so haven't removed it yet.

I guess this should instead refer to `DefaultValidationErrorFilter` and override `OnValidationException` (or `FormatFailure` even).

Reply:
sorry. i dont know enough about this codebase to work out what should be in this snippet. can u give me the code you would expect to be here?